stable/yourls/templates/deployment.yaml

```diff
@@ -0,0 +1,144 @@
+          image: "{{ .Values.image.registry }}/{{ .Values.image.repository }}:{{ .Values.image.tag }}"
+          imagePullPolicy: {{ .Values.image.pullPolicy | quote }}
+          env:
+            - name: ALLOW_EMPTY_PASSWORD
+            {{- if .Values.allowEmptyPassword }}
+              value: "yes"
+            {{- else }}
+              value: "no"
+            {{- end }}
+            - name: YOURLS_DB_HOST
+            {{- if .Values.mariadb.enabled }}
+              value: {{ template "mariadb.fullname" . }}:3306
+            {{- else }}
+              value: {{ .Values.externalDatabase.host | quote }}
+            {{- end }}
+            - name: YOURLS_DB_NAME
+            {{- if .Values.mariadb.enabled }}
+              value: {{ .Values.mariadb.db.name | quote }}
+            {{- else }}
+              value: {{ .Values.externalDatabase.database | quote }}
+            {{- end }}
+            - name: YOURLS_DB_USER
+            {{- if .Values.mariadb.enabled }}
+              value: {{ .Values.mariadb.db.user | quote }}
+            {{- else }}
+              value: {{ .Values.externalDatabase.user | quote }}
+            {{- end }}
+            - name: YOURLS_DB_PASS
+              valueFrom:
+                secretKeyRef:
+                {{- if .Values.mariadb.enabled }}
+                  name: {{ template "mariadb.fullname" . }}
+                  key: mariadb-password
+                {{- else }}
+                  name: {{ printf "%s-%s" .Release.Name "externaldb" }}
+                  key: db-password
+                {{- end }}
+            - name: YOURLS_DB_PREFIX
+              value: {{ .Values.yourlsTablePrefix | quote }}
+            - name: YOURLS_SITE
+              value: {{ .Values.yourlsSite | quote }}
+            - name: YOURLS_USER
+              value: {{ .Values.yourlsUsername | quote }}
+            - name: YOURLS_PASS
+              valueFrom:
+                secretKeyRef:
+                  name: {{ template "fullname" . }}
+                  key: yourls-password
+          {{- if .Values.extraEnv }}
+{{ toYaml .Values.extraEnv | indent 12 }}
+          {{- end }}
+          ports:
+            - name: http
+              containerPort: 80
+            - name: https
+              containerPort: 443
+          livenessProbe:
+            httpGet:
+              path: /admin/
+            {{- if not .Values.healthcheckHttps }}
+              port: http
+            {{- else }}
+              port: https
+              scheme: HTTPS
+            {{- end }}
+{{ toYaml .Values.livenessProbe | indent 10 }}
+          readinessProbe:
+            httpGet:
+              path: /admin/
+            {{- if not .Values.healthcheckHttps }}
+              port: http
+            {{- else }}
+              port: https
+              scheme: HTTPS
+            {{- end }}
+{{ toYaml .Values.readinessProbe | indent 10 }}
+          volumeMounts:
+            - mountPath: /apache
+              name: yourls-data
```

Comment:
@LeoColomb @darend are you sure that mounting the same volume into `/apache`, `/yourls`, `/php` is ok from security perspective?

Reply:
@goruha Got inspiration from the WordPress chart.

https://github.com/helm/charts/blob/3b0a8f9bb14e18f25022a0a51e41fee0020c2cf4/stable/wordpress/templates/deployment.yaml#L143-L153

Should I switch to a different volumes definition?

Reply:
@LeoColomb ok - let it be as is.